docs/iris/src/whatsnew/aggregate_directory.py

```diff
@@ -165,9 +235,23 @@ def compile_directory(directory, release):
     return compiled_text
 
 
-def run_compilation(release):
+def check_all_contributions_valid(release=None):
```

Comment:
It is used, by the new test.
I could move it there, but I'd rather not as I'd need to make some more bits public, and even then I think it is using implementation-specific assumptions.